test/jdk/java/lang/Thread/IsAlive.java

```diff
@@ -25,7 +25,7 @@
  * @test
  * @bug     8305425
  * @summary Check Thread.isAlive
- * @run main/othervm/timeout=10 IsAlive
+ * @run main/othervm/timeout=25 IsAlive
```

Comment:
Can it just run with the default timeout, meaning drop /timeout=N ? Also does this test need really need to run in othervm mode?